src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -959,6 +959,12 @@ public function update_source_name( $source_name ) {
 	 * @return array|WP_Error|Tribe__Events__Aggregator__Record__Queue
 	 */
 	public function process_posts( $data = array() ) {
+		if ( 'manual' === $this->type ) {
+			/** @var Tribe__Events__Aggregator__Service $service */
+			$service = tribe( 'events-aggregator.service' );
```

Comment:
Why two lines? Instead of just `tribe( 'events-aggregator.service' )->confirm_import( $this->meta )`